buildkite/scripts/build-release.sh

```diff
@@ -4,6 +4,11 @@ set -eo pipefail
 
 ([ -z ${DUNE_PROFILE+x} ] || [ -z ${MINA_DEB_CODENAME+x} ]) && echo "required env vars were not provided" && exit 1
 
+if [[ $1 = "archive_migration" ]] ; then
+    curl -O https://gist.githubusercontent.com/emberian/d1435ac373cd014a2c3054946bb6cd39/raw/caqti-upgrade.patch
```

Comment:
Is this a good idea 🤔 ?

Reply:
We can add one more level of hash to pin it to an exact patch.

Could also add the patch here to git.

Reply:
I think adding it here would be clearer. As this is to be fixed soon, I will stamp it to unblock the issue.